python/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_packages
+
+setup(
+  name='dex',
+  version='0.0.1',
+  description='A research language for typed, functional array processing',
+  license='BSD',
+  author='Adam Paszke',
+  author_email='apaszke@google.com',
+  packages=find_packages(),
+  package_data={'dex': ['libDex.so']},
+  install_requires=['numpy'],
```

Comment:
Install shouldn't require numpy right? It should only be a dependency.

Reply:
Looks like native_function.py imports numpy.

Also JAX used install_requires instead of a requirements.txt. Maybe this is fine?